Editor/Utility/SaveDuringPlay.cs

```diff
@@ -557,7 +551,6 @@ static void RestoreAllInterestingStates()
                     Undo.RegisterFullObjectHierarchyUndo(go, "SaveDuringPlay");
                     if (saver.PutFieldValues(go, roots))
                     {
-                        //Debug.Log("SaveDuringPlay: updated settings of " + saver.ObjetFullPath);
```

Comment:
actually, I'd leave this line in (but commented out).  Very useful for debugging, and I've had occasion to uncomment it several times